From 8a6c106790061a4b4b4c7a8ac8fd1b4a46e75250 Mon Sep 17 00:00:00 2001
From: Darren Etheridge <detheridge@ti.com>
Date: Fri, 18 Jul 2014 16:19:54 -0500
Subject: [PATCH 5/6] ARM: dts: am33xx: add DT node for gpu

Add the node into the am33xx.dtsi file for the SGX GPU
that is found in some variants of the SoC.

Signed-off-by: Darren Etheridge <detheridge@ti.com>
---
 arch/arm/boot/dts/am33xx.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/boot/dts/am33xx.dtsi b/arch/arm/boot/dts/am33xx.dtsi
index ebc3168..e26db94 100644
--- a/arch/arm/boot/dts/am33xx.dtsi
+++ b/arch/arm/boot/dts/am33xx.dtsi
@@ -881,6 +881,13 @@
 			reg = <0x48310000 0x2000>;
 			interrupts = <111>;
 		};
+
+		sgx@0x56000000 {
+			compatible = "ti,sgx";
+			ti,hwmods = "gfx";
+			reg = <0x56000000 0x1000000>;
+			interrupts = <37>;
+		};
 	};
 };
 
-- 
2.1.4

